process/blockProcessor.go

```diff
@@ -20,7 +20,9 @@ const (
 	internalMetaBlockByNoncePath  = "/internal/%s/metablock/by-nonce"
 	internalShardBlockByNoncePath = "/internal/%s/shardblock/by-nonce"
 
-	internalMiniBlockByHashPath = "/internal/%s/miniblock/by-hash"
+	internalMiniBlockByHashPath = "/internal/%s/miniblock/by-hash/%s/epoch"
+
+	internalStartOfEpochMetaBlockPath = "/internal/%s/startofepoch/metablock/by-epoch"
```

Comment:
can we add the epoch param in these strings as well?

Reply:
done :+1: